src/renderer/components/Execute.tsx

```diff
@@ -22,6 +23,23 @@ const Execute = () => {
     await window.connections.ipcRenderer.disconnect();
   };
 
+  const showAlert = () => {
+    return (
+      <Modal
+        show={alert}
+        onHide={() => setAlert(false)}
+        fullscreen
+        scrollable
+        animation={false}
+      >
+        <Modal.Header closeButton>
+          <Modal.Title>Procedure Code</Modal.Title>
+        </Modal.Header>
+        <Modal.Body className="code-wrapper">{code}</Modal.Body>
```

Comment:
Either this or add `white-space: break-spaces;` to `code-wrapper`
```suggestion
        <Modal.Body className="code-wrapper procedure-code">{code}</Modal.Body>
```